src/mpv.rs

```diff
@@ -96,19 +99,121 @@ unsafe impl GetData for i64 {
     }
 }
 
-unsafe impl SetData for i64 {
-    fn get_format() -> Format {
-        Format::Int64
+#[derive(Debug)]
```

Comment:
You could also derive `Hash` here.

Reply:
Or maybe not since the `Iterator` types are included… I guess for those `Hash` does not make much sense